drivers/mipi_dsi/Kconfig.renesas_ra

```diff
@@ -0,0 +1,13 @@
+# Renesas RA Family
+
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config MIPI_DSI_RENESAS_RA
+	bool "Renesas RA MIPI-DSI Host Controller"
+	default y
+	depends on DT_HAS_RENESAS_RA_MIPI_DSI_ENABLED
+	depends on DT_HAS_RENESAS_RA_DISPLAY_ENABLED
```

Comment:
Is this second dependency accurate? By this I mean can this MIPI DSI driver support command mode? If so, you probably *don't* need the parallel LCD controller driver to be enabled for all cases where you'd use the DSI driver.

Reply:
Done. I removed this depend on condition